Formula/percona-server.rb

```diff
@@ -1,25 +1,24 @@
 class PerconaServer < Formula
   desc "Drop-in MySQL replacement"
   homepage "https://www.percona.com"
-  url "https://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.24-27/source/tarball/percona-server-5.7.24-27.tar.gz"
-  sha256 "999593afa241660bc7860264b46a826faf97d80d411966fade24084f47b249a0"
-
-  bottle do
-    sha256 "79938125efb509b03ae247a23198c65fb6684eb49be9336860b5a92b56931b5b" => :mojave
-    sha256 "31d1745e94bc82f5fc8d2c677052a323f1dc76fdebe2369a6e23e583e48cc9ad" => :high_sierra
-    sha256 "1d6ad1ff891982cee38d7886df5dc99f907a802287a6bddb56c55c552dfcb430" => :sierra
-  end
+  url "https://www.percona.com/downloads/Percona-Server-8.0/Percona-Server-8.0.13-3/source/tarball/percona-server-8.0.13-3.tar.gz"
+  sha256 "ab00626775b85ea506c1d992d66acee50f0bfbde54b830731f61652e87ff93da"
 
   pour_bottle? do
     reason "The bottle needs a var/mysql datadir (yours is var/percona)."
     satisfy { datadir == var/"mysql" }
   end
 
   depends_on "cmake" => :build
+
+  # GCC is not supported either, so exclude for El Capitan.
```

Comment:
I'm not sure what you are trying to achieve with this line

Reply:
That line introduced in this [commit](https://github.com/Homebrew/homebrew-core/pull/27210/commits/86c3e0af755f67f39e77f9f8943616b7e0411917#diff-4d75be177e3055025e7bef3144311559R38), and I just copied from mysql formula.
Should I remove this lines?

Reply:
Yes please